package.json

```diff
@@ -3,14 +3,14 @@
     "version": "1.3.1",
     "description": "Boltz Exchange, Atomic Swaps, Web App",
     "scripts": {
-        "start": "cp src/configs/config.regtest.ts src/config.ts && vite",
-        "test": "cp src/configs/config.regtest.ts src/config.ts && jest --runInBand ./tests",
+        "test": "cp src/configs/regtest.json public/config.json && jest --runInBand ./tests",
```

Comment:
```suggestion
        "test": "npm run regtest && jest --runInBand ./tests",
```

Reply:
Step isnt necessary at all since test config is default: https://github.com/BoltzExchange/boltz-web-app/pull/507#discussion_r1527194777
But we could load the json file inside the tests aswell